### PR TITLE
Fix printf uppercase float conversion specifiers

### PR DESCRIPTION
**Pull request by GabrielRavier**
(more details in commit message)

### CODE REVIEW

File: test/libc/fmt/printf_uppercase_float_static_yoink_test.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*-*- mode:c;indent-tabs-mode:nil;c-basic-offset:2;tab-width:8;coding:utf-8 -*-│
+│vi: set net ft=c ts=2 sts=2 sw=2 fenc=utf-8                                :vi│
+╞══════════════════════════════════════════════════════════════════════════════╡
+│ Copyright 2021 Justine Alexandra Roberts Tunney                              │
```
Comment (G4Vi) [inline test/libc/fmt/printf_uppercase_float_static_yoink_test.c:4]: As this is your own test file, you can have the copyright for the file: Maybe  `Copyright 2023 Gabriel Ravier`

**Review by G4Vi — CHANGES_REQUESTED** [review]
The contents are good. That was a great idea to make sure yoinking the symbol is working as intended as one of steps of verifying the macro works as expected.

I can merge as is, but wanted to give you a chance to own the copyright, from `CONTRIBUTING.md`:  "ownership is much less of a concern in the unit test files so you're encouraged to put your copyright on those, provided it's ISC."

File: test/libc/fmt/printf_uppercase_float_static_yoink_test.c (hunk 1 above)
Comment (GabrielRavier) [reply test/libc/fmt/printf_uppercase_float_static_yoink_test.c]: I guess I'll take it, then, yeah.

**Comment by GabrielRavier** [comment]
Btw, whilst modifying the file to change the copyright notice, I've changed the way the tests are done slightly (mostly to make sure that each of the originally faulty conversion specifiers are yoinked when used by themselves, so I guess it'd be worth it to re-review just to be sure.

**Review by G4Vi — APPROVED** [review]
LGTM.

**Pull request merged**